less/bin-basket.less

```diff
@@ -0,0 +1,187 @@
+@bin-basket-circle-size: 150px;
+bin-basket {
+  cursor: pointer;
+  &.modern-basket {
+    >a {
+      background: @brand-primary;
+      color: white;
+      display: flex;
+      position: fixed;
+      right: 0;
+      top: 0;
```

Comment:
When the clerk menu is activated it overlaps with the op part of the basket dropdown in this case